### ISSUE TITLE
Without "DENO_TSCONFIG" config, an error occurs.

### ISSUE THREAD

**Issue by gnlow** [TooTallNate/vercel-deno#29]
I just followed the steps in [README](https://github.com/TooTallNate/vercel-deno/blob/f9d7ff0e03ab66e3bcc8d469074b6f3a400b2cff/README.md), but error below occured.
```
11:00:42.969  	Retrieving list of deployment files...
11:00:44.161  	Downloading 5 deployment files...
11:00:44.220  	Analyzing source code...
11:00:44.933  	Uploading build outputs...
11:00:46.032  	Installing build runtime...
11:00:47.455  	Build runtime installed: 1424.331ms
11:00:47.827  	Looking up build cache...
11:00:47.868  	Build cache not found
11:00:48.358  	/vercel/975ee1aa29c54514/.build-utils/.builder/node_modules/vercel-deno/dist/build.sh: line 13: DENO_TSCONFIG: unbound variable
11:00:48.360  	Error: Build script failed with exit code 1
11:00:48.360  	    at Object.build (/vercel/975ee1aa29c54514/.build-utils/.builder/node_modules/vercel-deno/dist/index.js:92:15)
11:00:48.360  	    at processTicksAndRejections (internal/process/task_queues.js:97:5)
11:00:48.360  	    at async buildStep (/var/task/sandbox-worker.js:21363:20)
11:00:48.360  	    at async mainSub (/var/task/sandbox-worker.js:21097:9)
11:00:48.360  	    at async main (/var/task/sandbox-worker.js:20960:5)
11:00:50.085  	Done with "api/hello.ts"
```
However, with `DENO_TSCONFIG` config, it was deployed well. (With empty `tsconfig.json`)  
**vercel.json**
```json
{
    "functions": {
        "api/**/*.[jt]s": { "runtime": "vercel-deno@0.6.3" }
    },
    "build": {
      "env": {
        "DENO_TSCONFIG": "api/tsconfig.json"
      }
    }
}
```
Is this intended?

**Comment by TooTallNate** [comment]
Shoot, that's definitely not intended. Thanks for pointing it out. Fixed in #30.

**Comment by TooTallNate** [comment]
Fixed in `vercel-deno@0.6.4`.
